src/v2/Apps/Settings/settingsRoutes.tsx

```diff
@@ -68,10 +68,10 @@ const SavesRoute = loadable(
       component.SettingsSavesRouteFragmentContainer,
   }
 )
-const SettingsRoute = loadable(
+const EditSettingsRoute = loadable(
   () =>
     import(
-      /* webpackChunkName: "settingsBundle" */ "./Routes/Settings/SettingsEditRoute"
+      /* webpackChunkName: "settingsBundle" */ "./Routes/EditSettings/SettingsEditSettingsRoute"
```

Comment:
This was infact unique (but poorly named) for whatever reason this route was literally importing and rendering another route inside of it.